manuscript/ha-docker-swarm/shared-storage-ceph.md

```diff
@@ -155,21 +155,24 @@ Grab the secret associated with the new user (you'll need this for the /etc/fsta
 ceph-authtool /etc/ceph/keyring.dockerswarm -p -n client.dockerswarm
 ```
 
+!!! note
+    These commands need to be executed from within the running `ceph-mon` container. To do this, we will use the `docker exec` command to open a shell *inside* the running container in order to run the commands above:
+    ```docker exec -it ceph-mon /bin/sh``` 
```

Comment:
```suggestion
    ```docker exec -it ceph-mon /bin/bash```
```

Please tell people to use a shell that support auto complete.

Reply:
`/bin/bash` doesn't seem to exist in the ceph containers, hence the use of `/bin/sh`.

Reply:
there is a bash in the container.

```
$ docker run --entrypoint /bin/bash ceph/daemon:latest-mimic
bash: warning: setlocale: LC_CTYPE: cannot change locale (en_US.UTF-8): No such file or directory
bash: warning: setlocale: LC_COLLATE: cannot change locale (en_US.UTF-8): No such file or directory
bash: warning: setlocale: LC_MESSAGES: cannot change locale (en_US.UTF-8): No such file or directory
bash: warning: setlocale: LC_NUMERIC: cannot change locale (en_US.UTF-8): No such file or directory
bash: warning: setlocale: LC_TIME: cannot change locale (en_US.UTF-8): No such file or directory
[root@77d7577f286b /]# ls
anaconda-post.log  bin  dev  etc  home  lib  lib64  media  mnt  opt  proc  root  run  sbin  srv  sys  tmp  usr  var
[root@77d7577f286b /]# ls /bin/bash
/bin/bash
[root@77d7577f286b /]#
```